zebra-consensus/src/block.rs

```diff
@@ -106,7 +109,7 @@ where
             // Field validity and structure checks
             let now = Utc::now();
             block.header.is_time_valid_at(now)?;
```

Comment:
I didn't spot it when making the issue that this PR closes but I think that this function should also move, for the same reason.  Moving it doesn't have to be in this PR though, it could be in a later one.

Reply:
The problem with that one is that it used inside `zebra_chain`. It means we will have to depend on consensus at chain, i have the strong impression we will not want to do that.

Reply:
Ah yeah, that's worth digging in to, let's not block this change on adding that one.